### DISCUSSION TITLE
size of window on startup

### DISCUSSION THREAD

**Discussion by F6Garfield** [mRemoteNG/mRemoteNG#2553] [Q&A] [answered]
Every time I start mremoteng the first thing I do is resize the window.
Is there a way in some config file to set the size of the window on startup ?

**Comment by FrankB-Wetropa** [comment]
Yes that would be great. 
I use different monitor configurations and resizing mRemote with open connections costs a lot of time when switching.
It would help, when using arguments like RDP-Client (mstsc.exe) uses (height, width and perhaps full window size).

**Reply by Kvarkas** [reply, marked as answer]
technicaly its should be in settings but 1.77.3 limited to save into settings as it will messing up with new way of doing that and could corrupt existing ones what still in use by stable version, so at this moment I do testing local db as local storage of such and very close to release a beta version (NB) to wider testing, what also include "remembering" session windows sizes.
